package.json

```diff
@@ -2,6 +2,7 @@
   "name": "fh-fhc",
   "description": "A Command Line Interface for FeedHenry",
   "version": "2.0.0-BUILD-NUMBER",
+  "minPlatformVersion": "7.15.0-1661",
```

Comment:
Does package.json allow arbitrary data in here?
Is there a property under which we can put this?